src/content/docs/en/guides/upgrade-to/v6.mdx

```diff
@@ -97,6 +97,30 @@ The following deprecated features are no longer supported and are no longer docu
 
 Some deprecated features may temporarily continue to function until they are completely removed. Others may silently have no effect, or throw an error prompting you to update your code.
 
+### `Astro` in `getStaticPaths()`
+
+Until now, it was possible to access the `Astro` object in `getStaticPaths()`. However, despite being typed the same as the `Astro` object accessible in the frontmatter, this object only had `site` and `generator` properties.
```

Comment:
```suggestion
In Astro 5.x, it was possible to access an `Astro` object inside `getStaticPaths()`. However, despite being typed the same as the `Astro` object accessible in the frontmatter, this object only had `site` and `generator` properties. This could lead to confusion about which `Astro` object properties were available inside `getStaticPaths()`. Additionally, warnings and error messages were often unclear.
```

Reply:
> Additionally, warnings and error messages were often unclear.

This is not exactly true I'd say: no warnings but if you got an error, that was because you'd try to do things on a value that is actually undefined

Reply:
Updated my comment. What about just cutting it off here?